charts/shoot-core/components/charts/monitoring/charts/blackbox-exporter/templates/deployment.yaml

```diff
@@ -48,6 +48,10 @@ spec:
         fsGroup: 65534
         supplementalGroups:
         - 1
+      {{- if semverCompare ">= 1.19.x" .Values.global.kubernetesVersion }}
```

Comment:
Nit: I see that `{{- if semverCompare ">= 1.19-0" .Capabilities.KubeVersion.GitVersion -}}` is used consistently in every other chart. You yourself have used it in the nginx charts. so maybe we could follow that throughout.
Also I think we have to use `1.19-0` instead of `1.19.x`, I remember some issue regarding this, but not sure. If you are sure it works for some version 1.19.x, then it's fine.